terraform/groups/provisioning-role/module-iam-policies/main.tf

```diff
@@ -0,0 +1,18 @@
+# Create policies
+resource "aws_iam_policy" "iam_policy_plan" {
+  name        = "payment-reconciler-terraform-plan"
```

Comment:
Policy names have to be region based at the moment, otherwise there will be a clash when run in a different region within same account.
In reality global resources like IAM should NOT be region-specific, IAM terraform state should be stored in a separate "global" s3 bucket.
This would reduce duplication of policies that only differ based on the region name (any region deployment of the service should be able to use the same IAM policy).
I think we need to prioritise this before fully implementing roles else we will be overrun with duplicated policies and roles.

Reply:
Hmmm, I see. Would this ever need to be run in a different region for the same account though? At the mo I have only one profile (development-eu-west-2) and not planning for any others in dev. Just one in Staging and one in Live to follow.

Reply:
I'll resolve the two comments below - and work from this one for all 3.